syncd/syncd_applyview.cpp

```diff
@@ -4,10 +4,14 @@
 #include "swss/logger.h"
 #include "swss/dbconnector.h"
 
+#include "CommandLineOptions.h"
+
 #include <inttypes.h>
 #include <algorithm>
 #include <list>
 
+extern std::shared_ptr<CommandLineOptions> g_commandLineOptions; // TODO move to syncd object
```

Comment:
>// TODO move to syncd object [](start = 65, length = 28)

What do you mean by `syncd object`? Can you just complete the task in this PR?

Reply:
no i cant, since im starting refactoring everything to object model, and this syncd object will be at the end of everything, since i can just like that switch everything without braking many things